packages/tooltip/demo/~patterns/stories/FocusStory.tsx

```diff
@@ -5,13 +5,16 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React from 'react';
+import React, { useRef } from 'react';
 import { StoryFn } from '@storybook/react';
 import classNames from 'classnames';
 import { useTooltip } from '@zendeskgarden/container-tooltip';
 
 export const FocusStory: StoryFn = () => {
-  const { getTooltipProps, getTriggerProps, isVisible, openTooltip, closeTooltip } = useTooltip();
+  const triggerRef = useRef<HTMLDivElement>(null);
+  const { getTooltipProps, getTriggerProps, isVisible, openTooltip, closeTooltip } = useTooltip({
+    triggerRef
+  });
```

Comment:
This story illustrates why `triggerRef` should be optional. Here, we had to create a placeholder ref just to satisfy the API, even though its functionality wasn't needed.

Reply:
Again, I don't agree for the reason stated previously – there's not a compelling reason to force the `containers` consumer to understand the why/if of supplying a `triggerRef`. Since there must always be a trigger, there's no reason not to require the ref here. Making it optional is a pattern departure from any of the other containers with cross-ref functionality – and therefore an undesirable approach.